tools/cldr-apps/js/src/views/InfoSelectedItem.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <a-tooltip :title="title">
+    <div class="info-selected-item" v-if="displayValue && description">
+      <div>
+        Value:
+        <span :class="valueClass" :lang="language" :dir="direction">{{
+          displayValue
+        }}</span>
+        <br />
+        <span class="value-description">{{ description }}</span>
+        <template v-if="linkUrl">
+          <br />
+          <a :href="linkUrl">{{ linkText }}</a>
+        </template>
+      </div>
+      <div v-html="testHtml"></div>
+      <div v-if="exampleHtml">
+        Example:<br />
+        <div
+          v-html="exampleHtml"
+          class="d-example"
+          :lang="language"
+          :dir="direction"
+        ></div>
+      </div>
+    </div>
+  </a-tooltip>
+</template>
+
+<script>
+import * as cldrText from "../esm/cldrText.mjs";
+
+export default {
+  data() {
+    return {
+      displayValue: null,
+      valueClass: null,
+      description: null,
+      language: null,
+      direction: null,
+      linkUrl: null,
+      linkText: null,
+      testHtml: null,
+      exampleHtml: null,
+      title: null,
+    };
+  },
+
+  created() {
+    this.title = cldrText.get("info_panel_selected");
+  },
+
+  methods: {
+    setValueAndClass(displayValue, valueClass) {
+      this.displayValue = displayValue;
+      this.valueClass = valueClass;
```

Comment:
It's great for now!  Once this is embedded with other vue components, we might want to use vue 3 props, to bind these values to something else (reactive instead of setters), but that's part of a future design discussion.